### DISCUSSION TITLE
Sonoff T1 EU opens no WLAN after flashing with tasmota 10

### DISCUSSION THREAD

**Discussion by dfoxg** [arendst/Tasmota#13699] [Q&A] [answered]
I´m trying to flash my new [Sonoff T1 EU](https://tasmota.github.io/docs/devices/Sonoff-T1/#sonoff-t1-eu-1ch2ch3ch-2018-10-03) with Tasmota 10. The flashing itself works without any problems, but no wlan is showing up (yes, i rebooted the device after flashing).

Output:
```
Command: esptool.py --port COM4 --baud 115200 --after no_reset write_flash --flash_size detect --flash_mode dio 0x00000 tasmota-DE_v10.0.0.bin --erase-all

esptool.py v3.0
Serial port COM4
Connecting....
Detecting chip type... ESP8266
Chip is ESP8285
Features: WiFi, Embedded Flash
Crystal is 26MHz
MAC: 3c:61:05:80:8c:06
Uploading stub...
Running stub...
Stub running...
Configuring flash size...
Auto-detected Flash size: 1MB
Erasing flash (this may take a while)...
Chip erase completed successfully in 2.9s
Flash params set to 0x0220
Compressed 578544 bytes to 398723...
Wrote 578544 bytes (398723 compressed) at 0x00000000 in 35.4 seconds (effective 130.7 kbit/s)...
Hash of data verified.

Leaving...
Staying in bootloader.

Firmware successfully flashed. Unplug/replug or reset device 
to switch back to normal boot mode.
```

To check if error is while flashing, i created a simple c++ programm with prints "hello world" -> flashing -> open serial monitor -> i can see hello world.

Has anyone an idea what is wrong?
Does tasmota write anything to the serial monitor? How can i check if tasmota is executed?

**Comment by sfromis** [comment, marked as answer]
You did not follow the flashing instructions when using flash mode dio. The recommendation from the project is tools designed to avoid errors by choosing "safe" options. When booting, the ESP8266 should produce outout, and shortly after that, also Tasmota, on 115200 baud.

**Reply by dfoxg** [reply]
Ah, thanks a lot!
